knip.jsonc

```diff
@@ -17,7 +17,13 @@
       ],
     },
     "packages/bench": {
-      "entry": ["vue-entry.js"],
+      "entry": [
+        "vue-entry.js",
+        "src/parallel-babel-plugin/impl.ts", // imported from src/parallel-babel-plugin/index.ts
+      ],
+      "babel": {
+        "config": "src/parallel-babel-plugin/impl.ts",
+      },
```

Comment:
This doesn't seems to be working. Is there a way to make knip detect these?
https://github.com/rolldown/rolldown/blob/1701888199dd59e5a219c0293c68dfc14cc66ea1/packages/bench/src/parallel-babel-plugin/impl.ts#L11-L14

Reply:
The exported value of `impl.ts` is not a Babel configuration, so Knip's Babel plugin will not handle it properly.

For now, let's ignore them:

```
"ignoreDependencies": ["@babel/preset-env", "@babel/preset-typescript"]
```

or something like this (regex)

```
"ignoreDependencies": ["@babel/preset-.+"]
```

Looks like `impl.ts` is not an unused file, so no need to add it as an `entry` I think?